makeflow/src/makeflow_amazon_batch_setup

```diff
@@ -0,0 +1,186 @@
+#!/bin/sh
+#
+#Copyright (C) 2018- The University of Notre Dame
+#This software is distributed under the GNU General Public License.
+#See the file COPYING for details.
+#
+
+
+load_fieldINI()
+{
+	grep $1 $2 | cut -d "=" -f 2 | tr -d ' '
+}
+
+load_fieldJSON()
+{
+	grep $1 $2 | cut -d ":" -f 2 | tr -d ' ' | tr -d ','
+}
+
+if [ -z "$1" ]; then
+	echo "Empty number of CPUs requested."
+	echo "Correct Ussage: ./makeflow_amazon_batch_setup [desired_num_cpus] [min_number_cpus] [max_number_cpus] (config-output)"
+	exit 32
+fi
+
+if [ -z "$2" ]; then
+	echo "Empty number of CPUs requested."
+	echo "Correct Ussage: ./makeflow_amazon_batch_setup [desired_num_cpus] [min_number_cpus] [max_number_cpus] (config-output)"
+	exit 33
+fi
+
+if [ -z "$3" ]; then
+	echo "Empty number of CPUs requested."
```

Comment:
Since you are checking for specific empty fields make the first line of each error more specific or just use a single catch all case for code clarity.